src/foundation.cs

```diff
@@ -140,7 +140,7 @@ namespace Foundation
 
 	delegate bool NSEnumerateErrorHandler (NSUrl url, NSError error);
 	delegate void NSMetadataQueryEnumerationCallback (NSObject result, nuint idx, ref bool stop);
-#if XAMCORE_4_0
+#if NET
 	delegate void NSItemProviderCompletionHandler (INSSecureCoding item, NSError error);
```

Comment:
I think we should keep the context on the item, according to [docs](https://developer.apple.com/documentation/foundation/nsitemprovidercompletionhandler?language=objc) it is the item to be loaded 
```suggestion
	delegate void NSItemProviderCompletionHandler (INSSecureCoding itemToBeLoaded, NSError error);
```

Reply:
https://github.com/xamarin/xamarin-macios/pull/13766